src/components/FieldCheckboxes/docs.stories.tsx

```diff
@@ -1,20 +1,172 @@
+import React, { useState } from 'react';
+
+import { Button, Divider, Flex, Stack } from '@chakra-ui/react';
 import { Formiz } from '@formiz/core';
 
-import { FieldCheckboxes } from './index';
+import {
+  FieldCheckboxes,
+  FieldCheckboxesCheckAll,
+  FieldCheckboxesItem,
+} from '.';
 
 export default {
   title: 'Fields/FieldCheckboxes',
 };
+
 export const Default = () => {
-  const options = [{ value: 'One' }, { value: 'Two' }, { value: 'Three' }];
+  const options = [
+    { label: 'Red', value: 'red' },
+    { label: 'Green', value: 'green' },
+    { label: 'Blue', value: 'blue' },
+  ];
+
+  return (
+    <Formiz onChange={console.log}>
+      <FieldCheckboxes
+        name="colors"
+        label="Colors"
+        required="You need to check at least one color"
+        options={options}
+      />
+    </Formiz>
+  );
+};
+
+export const IsDisabled = () => {
+  const options = [{ value: 'Red' }, { value: 'Green' }, { value: 'Blue' }];
+
   return (
     <Formiz>
       <FieldCheckboxes
-        name="FieldCheckboxes"
-        label="Label"
-        helper="Helper"
+        name="colors"
+        label="Colors"
+        isDisabled
         options={options}
       />
     </Formiz>
   );
 };
+
+export const WithDefaultValues = () => {
```

Comment:
The default value is not working. It blink when I click on the story.


https://user-images.githubusercontent.com/3920615/129027719-885c9ae3-0adf-4075-8ff2-ffa35e720d2f.mov

Reply:
An issue indeed, thank you for spotting it. I am working on a fix.